layouts/_default/article.html

```diff
@@ -1,6 +1,6 @@
 <div class="article ">
     <a class="block" href="{{ .Permalink }}">
-        <img src="{{ .Params.image | absURL }}" alt="" class="rounded-t-xl object-cover" style="height:212px; width:100%"></a>
+        <img loading="lazy" src="{{ .Params.image | absURL }}" alt="" class="rounded-t-xl object-cover" style="height:212px; width:100%"></a>
```

Comment:
Testing out using lazy load on the home page for images that appear below the fold.